### ISSUE TITLE
Currently hosted passband files have broken extinction tables

### ISSUE THREAD

**Issue by Raindogjones** [phoebe-project/phoebe2#911]
The passband files currently hosted on the passband server are either missing extinction tables or have extinction tables with incorrect values.

**Comment by aprsa** [comment]
I verified blackbody extinction tables thus far and everything seems to check out in the feature-blending branch. For example, this snippet works as expected:
```python
>>> import phoebe
>>> pb = phoebe.get_passband("Johnson:V")
>>> query_pts = np.ascontiguousarray([[10000, 3.1, 1.0]])
>>> pb.interpolate_extinct(query_pts, atm='blackbody')
array([[0.05699106]])
```
Note that the order of axes was flipped in the bug report: blackbody table axes are (teffs, rvs, ebvs).

**Comment by Raindogjones** [comment]
`pb.interpolate_extinct(Teff=10000.0,atm='blackbody',extinct=1.0,Rv=3.1)`
Still fails in 2.4.14, giving the error:
```python
Passband.interpolate_extinct(self, Teff, logg, abun, atm, extinct, Rv, photon_weighted)
   1953 nanmask = np.isnan(extinct_factor)
   1954 if np.any(nanmask):
-> 1955     raise ValueError('Atmosphere parameters out of bounds: atm=%s, extinct=%f, Rv=%f, Teff=%s, logg=%s, abun=%s' % (atm, extinct, Rv, Teff[nanmask], logg[nanmask], abun[nanmask]))
   1957 return extinct_factor

TypeError: 'float' object is not subscriptable
```

**Comment by aprsa** [comment]
Perhaps we should deprecate re-implementing extinction and use an [astropy-affiliated canned version](https://dust-extinction.readthedocs.io/en/latest/)?

**Comment by Raindogjones** [comment]
@aprsa That is definitely an option, but in the mean time the implementation in libphoebe should be fine. For blackbodies at least, it should just be a matter of flipping the way the tables are stored/queried in feature-blending to return to being backwards compatible.  For the other atmospheres, I'm not sure...

**Comment by Raindogjones** [comment]
And, in any case, the re-implementation in passbands.py will be pretty similar whether we use libphoebe or something from astropy - so that needs to be fixed either way.

**Comment by aprsa** [comment]
I vectorized all extinction computations that take computing time from ~1 day per passband to ~5 minutes per passband. Here's the tree:

https://github.com/aprsa/phoebe2/tree/vectorization_of_passband_computations

I've not yet committed the final vectorization for model atmospheres because I'm waiting for @Raindogjones's confirmation that everything makes sense; once I get his approval, I'll commit and this PR will be ready to go in.

**Comment by Raindogjones** [comment]
Tested using pb.interpolate_extinct() for blackbody, ck2004 and phoenix in v2.4.15 and everything seems to be working as expected. Thanks!

**Comment by aprsa** [comment]
[PR 972](https://github.com/phoebe-project/phoebe2/pull/972) vectorizes extinction computations, so this is now running again. The passbands will include only ck2004 and phoenix for the time being, until TMAP/Trembley issues have been resolved.

**Comment by aprsa** [comment]
All passbands have been recomputed. I will upload them to the server momentarily.

**Comment by aprsa** [comment]
I propose that we "clone" tables.phoebe-project.org API into staging.phoebe-project.org. I already created the apache rule on terra that currently points to a copy of the tables server. This staging server would then be used by us for testing any passband changes before we push it to production.

Separately, I tracked down and added the passbands that we supported in the past but that somehow fell through the cracks. We now have 70 supported passbands, including stock Johnson V and bolometric.
